src/main/ingestion-queues/queue.ts

```diff
@@ -51,48 +51,76 @@ export async function startQueue(
     try {
         if (server.KAFKA_ENABLED) {
             return await startQueueKafka(server, piscina, mergedWorkerMethods)
-        } else {
-            return startQueueRedis(server, piscina, mergedWorkerMethods)
         }
+        return startQueueRedis(server, piscina, mergedWorkerMethods)
     } catch (error) {
         status.error('💥', 'Failed to start event queue:\n', error)
         throw error
     }
 }
 
-function startQueueRedis(server: Hub, piscina: Piscina | undefined, workerMethods: WorkerMethods): Queue {
+function startQueueRedis(server: Hub, piscina: Piscina, workerMethods: WorkerMethods): Queue {
     const celeryQueue = new CeleryQueue(server.db, server.PLUGINS_CELERY_QUEUE)
 
+    // this queue is for triggering plugin jobs from the PostHog UI
     celeryQueue.register(
-        'posthog.tasks.process_event.process_event_with_plugins',
+        'posthog.tasks.plugins.plugin_job',
```

Comment:
Doesn't this now imply that `process_event_with_plugins` tasks on a non-kafka installation won't work?

Reply:
It does, but they are never sent in such a case. I think we only accepted them out of laziness/migration-tolerance, but it doesn't seem that ever was useful in practice.

Reply:
Ah, nevermind, it's **!**`!server.KAFKA_ENABLED` below.

Reply:
Yeah so before we just didn't run the celery queue at all if you had Kafka, as the plugin server only used it for events. 

Now we always run it, but don't listen to the events queue if Kafka is on.

This is git playing tricks on you as I didn't edit that line, I added a whole block of code that was based on the one that was there :D